runtime/compiletime/memory_instructions.c

```diff
@@ -1,78 +1,48 @@
 #include <assert.h>
 
-#include "types.h"
+#include "wasm_module_instance.h"
 
-uint32_t
+INLINE uint32_t
 instruction_memory_size()
 {
-	return local_sandbox_context_cache.memory.size / WASM_PAGE_SIZE;
+	return wasm_memory_get_page_count(current_wasm_module_instance.memory);
```

Comment:
`current_wasm_module_instance` is how we associate calls to the backing compiletime functions for wasm32 instructions with the running sandbox. I renamed this as "current_wasm_module_instance" since the only state that a wasm32 instruction should ever touch is the portion of the sandbox that aligns to a "wasm_module_instance." In the future, I imagine a sandbox struct will be refactored to contain a wasm_module_instance struct.